### PR TITLE
Minor Changes to #693

### PR DESCRIPTION
**Pull request by Denton-L**
Just cleaning up the documentation. Some of these changes may be a little subjective so please let me know if there is anything you may disagree with!

### CODE REVIEW

File: docs/control-structures.rst [hunk 1]
```diff
@@ -70,15 +70,15 @@ only (locally) sets the value and amount of gas sent with the function call and
...
-    just about anything. Be prepared that it calls into other contracts of
-    your system and perhaps even back into the calling contract before your
+    contract hands over control to the called contract and that may potentially
+    do just about anything. Be prepared in case it calls into other contracts of
+    your system or even back into the calling contract before the first
     call returns. This means
-    that the called contract can change state variables of the calling contract
-    via its functions. Write your functions in a way that e.g. calls to
-    external functions happen after any changes to state variables in your contract,
+    that the called contract can change the state variables of the calling contract
```
Comment (VoR0220) [inline docs/control-structures.rst]: leave this line as it was before.

File: docs/control-structures.rst [hunk 2]
```diff
@@ -70,15 +70,15 @@ only (locally) sets the value and amount of gas sent with the function call and
...
+    via its functions. Write your functions in a way that such that, for example, calls to
```
Comment (VoR0220) [inline docs/control-structures.rst]: in a way that, for example,

File: docs/security-considerations.rst [hunk 3]
```diff
@@ -168,30 +167,30 @@ about source code quality of course apply: Limit the amount of local variables,
 the length of functions and so on. Document your functions so that others
 can see what your intention was and whether it is different than what the code does.
 
-Use the Checks-Effects-Interactions pattern
+Use the Checks-Effects-Interactions Pattern
```
Comment (VoR0220) [inline docs/security-considerations.rst:171]: 👍

File: docs/security-considerations.rst [hunk 4]
```diff
@@ -111,26 +110,26 @@ Sending and Receiving Ether
...
-     means for the recipient to block progress in the sending contract. Again, the best practise here is to use
+     means for the recipient to block progress in the sending contract. Again, the best practice here is to use
      a "withdraw" pattern instead of a "send" pattern.
 
 Callstack Depth
 ===============
 
 External function calls can fail any time because they exceed the maximum
-call stack of 1024. In such situations, Solidity throws an exception.
+call stack of 1023. In such situations, Solidity throws an exception.
```
Comment (VoR0220) [inline docs/security-considerations.rst]: This is fine as before.

**Comment by Denton-L** [comment]
Thanks, @VoR0220!

File: docs/control-structures.rst [hunk 5]
```diff
@@ -70,15 +70,15 @@ only (locally) sets the value and amount of gas sent with the function call and
...
 
 .. warning::
-    Any interaction with another contract imposes a certain danger, especially
+    Any interaction with another contract imposes a potential danger, especially
     if the source code of the contract is not known in advance. The current
-    contract hands over control to the called contract and that might do
-    just about anything. Be prepared that it calls into other contracts of
-    your system and perhaps even back into the calling contract before your
+    contract hands over control to the called contract and that may potentially
+    do just about anything. Be prepared in case it calls into other contracts of
```
Comment (chriseth) [inline docs/control-structures.rst]: We should probably mention polymorphism here and contracts only adhering to an interface but that their implementation can do arbitrary other things.

**Comment by Denton-L** [comment]
Thanks, @chriseth, I added that in!

File: docs/security-considerations.rst [hunk 6]
```diff
@@ -6,22 +6,22 @@ While it is usually quite easy to build software that works as expected,
 it is much harder to check that nobody can use it in a way that was **not** anticipated.
 
 In Solidity, this is even more important because you can use smart contracts
-to handle tokens or even more valuable things. Furthermore, every execution of a smart
-contract happens in public as it is mostly open source.
+to handle tokens or, possibly, even more valuable things. Furthermore, every
+execution of a smart contract happens in public as it is mostly open environment.
```
Comment (chriseth) [inline docs/security-considerations.rst]: Hm, I think I throw together two concepts here and we should clarify that better. What about:
`... happens in public and, in addition to that, the source code is often available.`

File: docs/security-considerations.rst [hunk 7]
```diff
@@ -6,22 +6,22 @@ While it is usually quite easy to build software that works as expected,
...
 You can compare a smart contract with a web service that is open to the
-public (and thus also to malicous actors) and perhaps even open source.
+public (and thus, also to malicous actors) and perhaps even open source.
 If you only store your grocery list on that web service, you might not have
 to take too much care, but if you manage your bank account using that web service,
 you should be more careful.
 
 This section will list some pitfalls and general security recommendations but
-can of course never be complete. Also keep in mind that even if your
+can, of course, never be complete. Also, keep in mind that even if your
```
Comment (chriseth) [inline docs/security-considerations.rst:21]: Ah, the friendly comma ghost :-)

**Comment by chriseth** [comment]
Only the one comment, then ready to merge from my point of view.

**Comment by Denton-L** [comment]
Alright, done!

**Pull request merged**